src/main/java/io/stargate/sgv2/jsonapi/exception/playing/DatabaseException.java

```diff
@@ -0,0 +1,25 @@
+package io.stargate.sgv2.jsonapi.exception.playing;
+
+public class DatabaseException extends ServerException {
+
+  public static final Scope SCOPE = Scope.DATABASE;
+
+  public DatabaseException(ErrorInstance errorInstance) {
+    super(errorInstance);
+  }
+
+  public enum Code implements ErrorCode<DatabaseException> {
+    FAKE;
+
+    private final ErrorTemplate<DatabaseException> template;
+
+    Code() {
+      template = ErrorTemplate.load(DatabaseException.class, FAMILY, SCOPE, name());
```

Comment:
Loading from Enum instance constructors is extremely fragile -- I think it happens at class loading time -- and could cause really gnarly failure modes.

But then again, as long as our test coverage is good, maybe that's not a big issue.

Reply:
Ohhhh. Much worse -- this is called once per each Enum value? Does it really try to load something, or is that misnamed method? (to me, loading means reading YAML config file).

Reply:
The ErrorTemplate will cause the ErrorConfig to load, which happens once and loads all of the YAML config at once and only once. 

Then this call to load() will fail if there is an error code defined in code but not in the yaml, so we want that failure to happen.